shared/src/assets/icons/erc20Assets.tsx

```diff
@@ -306,6 +306,30 @@ export const WAVAXLogo: React.FC<SVGProps & { showBackground?: boolean }> = ({
   </svg>
 );
 
+export const SAVAXLogo: React.FC<SVGProps & { showBackground?: boolean }> = ({
+  showBackground = false,
+  ...props
+}) => (
+  <svg
+    viewBox="0 0 56 56"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    {showBackground && (
+      <path
+        d="M28 56C43.464 56 56 43.464 56 28C56 12.536 43.464 0 28 0C12.536 0 0 12.536 0 28C0 43.464 12.536 56 28 56Z"
+        fill="#11BAF0"
+        fill-opacity="0.12"
```

Comment:
We need to change `fill-opacity` to `fillOpacity`

Reply:
Fixed - good catch